configs/stash-origin/tmpfiles/stash-origin.conf

```diff
@@ -1,3 +1,3 @@
 # The auto-generated authfile will live here
-d /run/stash-origin 0775 xrootd xrootd
-d /run/stash-origin-auth 0775 xrootd xrootd
+d /run/stash-origin 0755 xrootd xrootd
+d /run/stash-origin-auth 0755 xrootd xrootd
```

Comment:
Are we sure this is safe? Do we know if anything relies on the group membership? Though I'm guessing it's fine since the spec file just drops them down 0755

Reply:
Well, 0755 is what they get set to on install, it's just that the tmpfiles conf sets them to something different on a reboot. Since stashcache works for me just after installing, I assume 0755 is good